10/streams/developer-guide/manage-topics.html

```diff
@@ -84,7 +84,7 @@
       <p>
         Kafka Streams automatically creates internal repartitioning and changelog
         topics, and does <em>not</em> use broker auto-topic creation.
-        Therefore, internal topics take on the default broker configurations for
+        Therefore, internal topics do not take on the default broker configurations for
```

Comment:
Thinking about this once again, some broker defaults are actually used... Just not all. Maby best to rephrase as `do not take on [all] the default`?

Might be worth to clarify which configs? In particular number of partitions and replication factor, as well as, cleanup strategy (and others -- depending on the version, but maybe calling out those three is sufficient?)

Reply:
Good thought to be more accurate. I'll try to digest what configs are not respecting broker defaults, which could be changed between versions. Otherwise, might just follow your suggestion. Thanks

Reply:
Couldn't find details more accurate than those you mentioned. I am happy to apply what you suggested. @mjsax